internal/binutils/binutils.go

```diff
@@ -179,10 +179,25 @@ func (b *binrep) openMachO(name string, start, limit, offset uint64) (plugin.Obj
 	}
 	defer of.Close()
 
+	// Subtract the load address of the __TEXT section. Usually 0 for shared
+	// libraries or 0x100000000 for executables. You can check this value by
+	// running `objdump -private-headers <file>`.
+
+	textSegment := of.Segment("__TEXT")
+	if textSegment == nil {
+		return nil, fmt.Errorf("Parsing %s: no __TEXT segment", name)
```

Comment:
Error messages should start with lowercase: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Suggesting `fmt.Errorf("not a mach executable: no __TEXT segment: %s", name)`, to be somewhat consistent with `fmt.Errorf("unrecognized binary: %s", name)` above in this file.

Reply:
I just copied the one above but ok.